docs/hipchat

```diff
@@ -1,6 +1,5 @@
-Note: HipChat has a new and improved integration with GitHub. While this
-service will continue to work, you really should check out the new integration.
-Read more here: http://help.hipchat.com/knowledgebase/articles/64389-github-integration
+Note: Atlassian provides its own native integration for GitHub.
+Read more here: https://hipchat.com/addons/github-for-hipchat
```

Comment:
Could we keep the text noting that we prefer the use of Atlassian's integration and not the hipchat service provided here?